<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>142</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>137</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnNormal">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>普通</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMulti">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>多个标准按钮</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnInfo">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWarnning">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>warnning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnQuestion">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>question</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCritical">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>critical</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAbout">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>About</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAboutQt">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>关于Qt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpenFileName">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>OpenFileName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSaveFileName">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SaveFileName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnInputDialog">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>InputDialog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnProgressDialog">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>191</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ProgressDialog</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>137</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>190</width>
       <height>85</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
